README.md

```diff
@@ -175,3 +175,14 @@ gcloud config list, to display current account
 ```
 
 `gcloud app deploy` does not support `--update-env-vars RELEASE=$RELEASE` like `gcloud run deploy` does with Cloud Run
+
+## Local Run with AI Suggestions
+
+1. Add your OPENAI_API_KEY= to flask .env (personal one ATM)
+2. Start flask locally (./deploy.sh --env=local flask)
+3. Take note of where it's running (likey http://127.0.0.1:8080)
+4. Change NEXT_PUBLIC_FLASK_BACKEND to the url from Step 3
+5. Run the next server (npm run dev)
```

Comment:
`./deploy.sh react flask` will do these steps automatically for you

Reply:
Nice! Looks like running `./deploy.sh flask next` gives me this backend url which fails

```
Supported backend types:  {
  flask: 'http://localhost:8080',
```

but running them separately correctly assigns it and it succeeds.

Reply:
@lukemun this is likely because NEXT_LOCAL_PORT is missing from your `env-config/local.env`

please also update these two:
* https://github.com/sentry-demos/empower/blob/master/env-config/example.env 
* https://github.com/sentry-demos/empower-config/blob/main/local.env (separate repo)

Reply:
Adding to example.env, will put up another CR for local.env

Reply:
Upon another look I don't think this is necessary, sorry I got confused earlier. See my slack message

Reply:
Add the param from slack and got it working